File: MercaderSG/Comercial/NotaPedido/UltimaNPRepor.vb
Imports Negocios
Imports Servicios
Imports Entidades
Imports Excepciones
Imports CrystalDecisions.CrystalReports
Imports CrystalDecisions.CrystalReports.Engine

Public Class UltimaNPRepor

    Private Sub UltimaNPRepor_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        Me.Text = My.Resources.ArchivoIdioma.UltimaNotaPedidoFrm

        Dim NPDS As New GeneralDS

        NotaPedidoRN.CargarUltimaNotaPedido(NPDS)

        Dim Reporte As New NotaPedidoRP
        Reporte.SetDataSource(NPDS)
        UltimaNotaPedRPV.ReportSource = Reporte
    End Sub
End Class

File: MercaderSG/Comercial/NotaPedido/UltimaNPRepor.cs
using System;
using Negocios;

namespace MercaderSG
{
    public partial class UltimaNPRepor
    {
        public UltimaNPRepor()
        {
            InitializeComponent();
        }

        private void UltimaNPRepor_Load(object sender, EventArgs e)
        {
            Text = My.Resources.ArchivoIdioma.UltimaNotaPedidoFrm;
            var NPDS = new GeneralDS();
            NotaPedidoRN.CargarUltimaNotaPedido(NPDS);
            var Reporte = new NotaPedidoRP();
            Reporte.SetDataSource(NPDS);
            UltimaNotaPedRPV.ReportSource = Reporte;
        }
    }
}